tests/unit/api/test_user.py

```diff
@@ -28,6 +28,22 @@ def test_gets(self) -> None:
         self.assertEqual(list(response)[1], HTTPStatus.OK)
         self.mock_client.get_users.assert_called_once()
 
+    @mock.patch('metadata_service.api.request')
+    def test_post(self, mock_request: MagicMock) -> None:
+        mock_request.get_json.return_value = [dict(email='username@gmail.com', is_active=True)]
```

Comment:
tiny nit: maybe example.com so it won't resolve to any real user?

Reply:
fair enough. I updated to use example.com email addresses